engine/classes/Elgg/Application.php

```diff
@@ -110,6 +106,14 @@ public function __construct(ServiceProvider $services) {
 	}
 
 	/**
+	 * Returns current session
+	 * @return \ElggSession
+	 */
+	public static function session() {
+		return self::$_instance->_services->session;
```

Comment:
Benefit over elgg_get_session()?

Reply:
Trying to get session before core files are loaded.

Reply:
I will remove, we can add back if needed.